parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -377,11 +376,9 @@ ensureArtifactStreaming() {
   sudo /opt/acr/tools/overlaybd/config.sh exporterConfig.port 9863
   modprobe target_core_user
   curl -X PUT 'localhost:8578/config?ns=_default&enable_suffix=azurecr.io&stream_format=overlaybd' -O
-  systemctl enable /opt/overlaybd/overlaybd-tcmu.service
-  systemctl enable /opt/overlaybd/snapshotter/overlaybd-snapshotter.service
-  systemctl start overlaybd-tcmu
-  systemctl start overlaybd-snapshotter
-  systemctl start acr-nodemon
+  systemctlEnableAndStart /opt/overlaybd/overlaybd-tcmu.service 30
+  systemctlEnableAndStart /opt/overlaybd/snapshotter/overlaybd-snapshotter.service 30
+  systemctlEnableAndStart acr-nodemon 30
```

Comment:
Previously, I just had acr-nodemon starting (with a systemctl enable). I discovered two issues from the extended artifact streaming e2e I'd added -- acr-nodemon was not resilient to reboots and it occasionally timed out, causing a failure. This will provide it an explicit timeout value.